src/realm-settings/TokensTab.tsx

```diff
@@ -402,6 +403,32 @@ export const RealmSettingsTokensTab = ({
               )}
             />
           </FormGroup>
+          <FormGroup
+            label={t("defaultAudValue")}
+            fieldId="defaultAudValue"
+            id="default-aud-value-label"
+            labelIcon={
+              <HelpItem
+                helpText="realm-settings-help:defaultAudValueTooltip"
+                fieldLabelId="realm-settings:defaultAudValue"
+              />
+            }
+          >
+            <Controller
+              name="defaultAudValueForAccessToken"
+              defaultValue={""}
+              control={form.control}
+              render={({ onChange, value }) => (
+                <KeycloakTextInput
+                  id="default-aud-value"
```

Comment:
The id here should match the `fieldId` of the encapsulating `FormGroup`.
```suggestion
                  id="defaultAudValue"
```